run-local-testnet/action.yaml

```diff
@@ -0,0 +1,97 @@
+name: Run Local Testnet
+description: Use the Aptos CLI to run a local testnet. Once this action is succeeds, a local testnet will be running and ready to use.
+
+inputs:
+  PNPM_VERSION:
+    description: "The version of pnpm to install."
+    default: 8.15.6
+  NODE_VERSION:
+    description: "The version of node to install. This must be specified if a .node-version file is not present."
+    required: false
+  CLI_GIT_REF:
+    description: "The git ref (e.g. a tag, like 'testnet' or a commit SHA) of the Aptos CLI to use. If not given, we will use the latest released CLI. If given, the local testnet is run from a Docker image."
+    required: false
+  DOCKER_ARTIFACT_REPO:
+    description: "The GCP Docker artifact repository + user. Only used if CLI_GIT_REF is set."
+    default: "docker.io/aptoslabs"
+    required: false
```

Comment:
I'm not sure if you're meant to set required to true or false if there is a default.

Reply:
Well for workflow_call at least it should be false, otherwise you have to set it no matter what, essentially making the default useless.